### PR TITLE
Add example Helm chart

### PR DESCRIPTION
**Pull request by heimweh**
This adds an example helm chart that can be used to get up and running with the boring registry.

### CODE REVIEW

File: helm/boring-registry/values.yaml [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+    repository: ghcr.io/tiermobility/boring-registry
+    tag: v0.7.0
+    pullPolicy: Always
+
+nameOverride: ""
+fullnameOverride: ""
+
+server:
+  # If set to true, the server component will be created.
+  enabled: true
```
Comment (oliviermichaelis) [inline helm/boring-registry/values.yaml:17]: Do we have a use-case for not enabling the server component?

**Review by oliviermichaelis — COMMENTED** [review]
Looks good to me, I think it would make sense to merge it soon and get some experience with it.
We could also publish the helm chart to ghcr.io afaik to make it publically available

File: helm/boring-registry/values.yaml (hunk 1 above)
Comment (heimweh) [reply helm/boring-registry/values.yaml:17]: Probably not, happy to remove it :)

**Review by oliviermichaelis — APPROVED** [review]
LGTM :+1: Thanks @heimweh

**Pull request merged**